modules/apps/frontend-js/frontend-js-minifier/src/main/java/com/liferay/frontend/js/minifier/internal/GoogleJavaScriptMinifier.java

```diff
@@ -72,7 +73,13 @@ public String compress(String resourceName, String content) {
 				SourceFile.fromCode("extern", StringPool.BLANK), sourceFile,
 				compilerOptions);
 
-			if (compiler.hasErrors()) {
+			boolean ignoredFile = Stream.of(
```

Comment:
Mmm, IINM, we don't use Stream for these things because a plain for loop is much faster...